stdlib/http/src/main/ballerina/http/resiliency/http_circuit_breaker.bal

```diff
@@ -551,6 +551,7 @@ function updateCircuitState(CircuitHealth circuitHealth, CircuitState currentSta
                             CircuitBreakerInferredConfig circuitBreakerInferredConfig) returns (CircuitState) {
     lock {
         CircuitState currentState = currentStateValue;
+        io:println(circuitHealth);
```

Comment:
This is not relevant to optional field change. Added as an improvement?

Reply:
Ah yes.. This should not have been committed. Will remove.